src/components/admin/article/components/CategorySelector.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import _ from 'lodash';
+
+import BaseComponent from 'lib/BaseComponent';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+
+export default class Component extends BaseComponent {
```

Comment:
This could be a generalised component, i.e. a list selector, instead of a category selector.